tests/conftest.py

```diff
@@ -17,8 +17,22 @@
 }
 
 
+@pytest.fixture(scope="session")
+def pulp_cli_settings(tmp_path_factory):
+    """
+    This session fixture will setup the config file once.
+    Use trough `pulp_cli_env` in order to setup XDG_CONFIG_HOME
```

Comment:
Is this suppose to say "Use through"?

Reply:
Yes, but as it does not seem too obvious to you, let me rephrase it.